app/assets/javascripts/discourse/config/deprecation-workflow.js

```diff
@@ -11,5 +11,9 @@ globalThis.deprecationWorkflow.config = {
     },
     { handler: "silence", matchId: "discourse.select-kit" },
     { handler: "silence", matchId: "discourse.d-section" },
+    {
+      handler: "silence",
+      matchId: "discourse.decorate-widget.hamburger-widget-links",
+    },
```

Comment:
We still have tests for this deprecation we want to keep so this is need to prevent the deprecation from raising an error in dev which we do as a default.